scripts/lib/build-utils.ts

```diff
@@ -177,6 +179,17 @@ export default {
         new webpack.EnvironmentPlugin({ NODE_ENV: mode }),
       ],
     };
+
+    if (this.isDevelopment) {
+      config.plugins ??= [];
+      config.plugins.push((new ForkTsCheckerWebpackPlugin() as unknown) as webpack.EnvironmentPlugin);
+      config.plugins.push((new ForkTsCheckerNotifierWebpackPlugin({
```

Comment:
Do we really need to pull in a dependency to do notifications (growl-style) we don't really need?

Reply:
Without the notifier the time between running `yarn dev` and seeing the window increases from about 60 seconds to 114.

Reply:
Wow, okay, that's totally not obvious from the [package documentation](https://www.npmjs.com/package/fork-ts-checker-notifier-webpack-plugin) — it sounded like it was just providing extra notification.